client/src/pages/ProfilePage/ProfilePage.js

```diff
@@ -20,23 +22,167 @@ export default function ProfilePage() {
 
-  function handleDelete(tagIndex) {
-    const newTags = tags.filter((tag, i) => i !== tagIndex);
-    setTags(newTags);
+  const [dropDownOptions, setDropDownOptions] = React.useState([]);
+
+  function setProfileVariables(coop) {
+    setCoop(coop);
+    setLocation(coop['addr']);
+    setPhone(coop['phone_number']);
+    setTags(coop['tags']);
+    setMission(coop['mission_statement']);
+    setDescription(coop['description_text']);
+    setFbLink(coop['fb_link']);
+    setInstaLink(coop['insta_link']);
+    setEmail(coop['email']);
+    setWebsite(coop['website']);
+    setProfilePicture(coop['profile_pic']);
+    setName(coop['coop_name']);
   }
 
   React.useEffect(() => {
     async function fetchData() {
       try {
         const res = await axios.get('/api/coop');
         setProfileVariables(res.data);
+
+        function setDefaultTags(coop) {
+          setTagsRole(coop['tags'].map(tag => makeTagDictionary(tag)));
+        }
+
+        setDefaultTags(res.data);
+
+        const allTags = await axios.get('/api/tags');
+        setDropDownOptions(allTags.data);
       } catch (err) {
         setRedirect(true);
       }
     }
     fetchData();
   }, []);
 
+  const [open, setOpen] = React.useState(false);
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+    setTags(tagsRole.map(dict => dictValue(dict)));
+  };
+
+  const locationOptions = [
+    { value: 'Alabama', label: 'Alabama' },
+    { value: 'Alaska', label: 'Alaska' },
+    { value: 'Arizona', label: 'Arizona' },
+    { value: 'Arkansas', label: 'Arkansas' },
+    { value: 'California', label: 'California' },
+    { value: 'Colorado', label: 'Colorado' },
+    { value: 'Connecticut', label: 'Connecticut' },
+    { value: 'Delaware', label: 'Delaware' },
+    { value: 'District of Columbia', label: 'District of Columbia' },
+    { value: 'Florida', label: 'Florida' },
+    { value: 'Georgia', label: 'Georgia' },
+    { value: 'Hawaii', label: 'Hawaii' },
+    { value: 'Idaho', label: 'Idaho' },
+  ];
+  const raceOptions = [
+    { value: 'black', label: 'Black-owned' },
+    { value: 'native', label: 'Native-owned' },
+    { value: 'asian', label: 'Asian-owned' },
+    { value: 'pacificIslander', label: 'Pacific Islander-owned' },
+    { value: 'hispanicOrLatinx', label: 'Hispanic or Latinx-owned' },
+  ];
+  const productOptions = [
+    { value: 'fruits', label: 'Fruits' },
+    { value: 'vegetables', label: 'Vegetables' },
+    { value: 'natural', label: 'Natural Goods' },
+    { value: 'wholefoods', label: 'Wholefoods' },
+    { value: 'dairy', label: 'Dairy' },
+    { value: 'brew', label: 'Brew' },
+    { value: 'fish', label: 'Fish' },
+    { value: 'meats', label: 'Meats' },
+  ];
+  const otherOptions = [
+    { value: 'nonGMO', label: 'Verified Non-GMO' },
+    { value: 'startup', label: 'Startup' },
+    { value: 'queer', label: 'Queer-owned' },
+    { value: 'nonprofit', label: 'Non-profit' },
+  ];
```

Comment:
if we're not using these, can we just remove that them so they're not confusing? just have the other dropdowns not have any options available